src/main/java/Deadlines.java

```diff
@@ -0,0 +1,61 @@
+package packagedirectory.test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class Deadlines extends Tasks {
```

Comment:
I think you did a great job in forming the String output clearly first before printing it. Just a minor suggestion, I think you could consider using 'Deadline' instead of 'Deadlines' and apply it to other classes that may not be in right form (singular/plural). Otherwise, great usage of inheritance as well!